Contest_API.md

```diff
@@ -2064,7 +2064,7 @@ Properties of problem data objects:
 | problem\_id  | ID        | Identifier of the [ problem](#problems).
 | num\_judged  | integer   | Number of judged submissions (up to and including the first correct one),
 | num\_pending | integer   | Number of pending submissions (either queued or due to freeze).
-| solved       | boolean ? | Required iff contest:scoreboard_type is `pass-fail`.
+| solved       | boolean   | Required iff contest:scoreboard_type is `pass-fail`.
```

Comment:
If the scoreboard type is "pass-fail", then this always has a well-defined value, so let's be explicit about it and not allow `null`.

Reply:
But it's currently not possible to state in `/access` that you are not providing this.

Reply:
That is a good point, but let's try to see if we can add that. It feels a bit weird that here and also for `scoreboard.rows.problems` we have it nullable, just because we can't specify it in `/access`.

Reply:
Agreed.